src/main/scala/org/pac4j/http4s/CallbackService.scala

```diff
@@ -0,0 +1,31 @@
+package org.pac4j.http4s
+
+import org.http4s.{Request, Response}
+import org.pac4j.core.config.Config
+import org.pac4j.core.engine.DefaultCallbackLogic
+import org.pac4j.core.http.adapter.HttpActionAdapter
+import scalaz.concurrent.Task
+
+/**
+  * Http4s Service to handle callback from Id Provider
+  */
```

Comment:
Just my 2 cents here but I like to see the `@author` and `@since` annotations in the Javadoc of the classes

Reply:
Added Author. Left out since as it is the initial version.